### ISSUE TITLE
Nvidia runtime (--runtime=nvidia)

### ISSUE THREAD

**Issue by gosha20777** [microsoft/Docker.DotNet#413]
I want to run docker with this parametr
`runtime=nvidia`

E.g. cli command is:
`docker run --rm --runtime=nvidia nvidia/cuda:9.0-devel nvcc --version`

Now I sucsessfully running my service via dotnet
Its equvalent this command:
`docker run -p 5000:5000 gosha20777/my-service:1.2`

But I want to run also it with nvidia docker (nvidia runtime)
`docker run --runtime=nvidia -p 5000:5000 gosha20777/my-gpu-service:1.2`

How I can do it?

**Comment by mocsharp** [comment]
Is there any way to specify `runtime`?

**Comment by HofmeisterAn** [comment]
I think you’re looking for:

https://github.com/dotnet/Docker.DotNet/blob/13ad3a144d24af0451d5a1c5f749a92e1252888f/src/Docker.DotNet/Models/HostConfig.Generated.cs#L148

**Comment by mocsharp** [comment]
> I think you’re looking for:
> 
> https://github.com/dotnet/Docker.DotNet/blob/13ad3a144d24af0451d5a1c5f749a92e1252888f/src/Docker.DotNet/Models/HostConfig.Generated.cs#L148

Yes! Thank you @HofmeisterAn, I was about to post an update with the finding 😂.
However, I couldn't find options for `--gpus`. Any idea where to find it? 🙏

**Comment by mocsharp** [comment]
@HofmeisterAn Big thanks!!

**Comment by hillin** [comment]
I think what @HofmeisterAn actually mean is 
https://github.com/dotnet/Docker.DotNet/blob/f58748616cc5b679b25496926c5688294c94d850/src/Docker.DotNet/Models/HostConfig.Generated.cs#L211
